README.md

```diff
@@ -1,2 +1,36 @@
 # django-ilmoitin
-Django app for sending notifications
+
+A templated Django messaging library:
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django-ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. Create a `notifications.py` file in django app.
+
+```python
+from django_ilmoitin.registry import notifications
+
+notifications.register("event_created", "Event created") # Register event
+```
+
+2. Import notifications in your apps.py:
```

Comment:
1. `apps.py`:
2. please, give a bigger example of the `AppConfig`, so that it would be even more clear where does this code go.